website/api/manager/ManagerAPI.php

```diff
@@ -65,6 +65,13 @@ private function getTrueskillMatchQuality($rankingValues) {
         return floatval($lines[0]);
     }
 
+    private function clearPairing() {
```

Comment:
Yeah, that makes sense. It just got factored out into a separate function just because the game api function can exit (return) from two code paths, each of which need to clear the pairing.